packages/opentelemetry-api/src/api/propagation.ts

```diff
@@ -96,6 +96,16 @@ export class PropagationAPI {
     return this._getGlobalPropagator().extract(context, carrier, getter);
   }
 
+  /**
+   * Return a list of all fields which may be used by the propagator.
+   *
+   * This list should be used to clear fields before calling inject if a carrier is
+   * used more than once.
+   */
```

Comment:
Nit: add an example how to do it

Reply:
Thanks.
Actually, I just copied the comment from [TextMapPropagator](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L68)

Reply:
Actually I think it should probably be removed from both places. The functions just list which fields may be used. How the output is used is up to the user and there may be additional use-cases we may not have thought of.

Reply:
@dyladan I agree with you, will update the PR.
My use case is different: I use message attributes to propagate context in sqs, and need to add the message attributes names when calling sqs `receiveMessage` for them to be sent back with the message.